schema/src/main/java/se/digg/dgc/valueset/v1/validation/ValueSetValidationResult.java

```diff
@@ -157,7 +159,7 @@ public String getMessage() {
   @Override
   public String toString() {
     return String.format("ValueSetValidationResult [result=%s, propertyName='%s', message='%s', children=%s]",
-      this.result, this.propertyName, this.message, this.children);
+      this.result, this.propertyName, this.message, this.children.size());
```

Comment:
Det blir lite kaka på kaka om vi tar med children eftersom meddelanden aggregeras

Reply:
Yes. Change the "children=%s" to "children=%d".